pages/result.js

```diff
@@ -84,153 +111,180 @@ export default function Index() {
           )}
         </Box>
       </Box>
-      <Box component="div" mt={1}>
-        {battles &&
-          sortedTopThree.map((data, index) => {
-            return (
-              <Box
-                alignItems="center"
-                sx={{
-                  display: 'flex',
-                  bgcolor: 'background.paper',
-                  p: 2,
-                  mb: index !== 2 && 1,
-                  borderRadius: 1,
-                  height: '33%',
-                }}
-                key={data.id}
-              >
-                <Badge
-                  color="primary"
-                  badgeContent={
-                    <Typography sx={{ fontSize: '1.2rem', fontWeight: 600 }} component="span">
-                      {index + 1}
-                    </Typography>
-                  }
-                  anchorOrigin={{
-                    vertical: 'top',
-                    horizontal: 'left',
-                  }}
-                >
-                  <Avatar alt={data.name} src={data.picture} sx={{ width: 60, height: 60 }} />
-                </Badge>
+      <Container sx={{paddingY: 4}} ref={containerRef}>
+        <Typography variant="h4" component="h4" align="center" sx={{ fontWeight: 300, textTransform: 'uppercase', marginBottom:2 }}>
+            Your Results! 
+        </Typography>
+        
```

Comment:
I think the container should have wrapped the filters too,

so that when the result is shared with other users, they can see what's the result filter based on